net/openvpn/files/openvpn.init

```diff
@@ -140,10 +140,11 @@ openvpn_get_credentials() {
 openvpn_add_instance() {
 	local name="$1"
 	local dir="$2"
-	local conf="$3"
+	local conf=$(basename "$3")
```

Comment:
great! but it is good to use `$dir/$conf` since it is name not path in var `conf`
so basename is not needed, I think.

Reply:
oops. looks like you are correct. since there is case where conf is absolute path.